packages/viewer-sandbox/tsconfig.json

```diff
@@ -10,7 +10,7 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "esModuleInterop": true,
-    "noEmit": true,
+    // "noEmit": true,
```

Comment:
why?

Reply:
Because I don't know how rollup works, and thought that since it's going to bundle up everything in the end, there would be no need for any standalone compiled js